holmscan/cli.py

```diff
@@ -259,6 +263,34 @@ def run(cli_args, sub_args):
                         **tabulate_args
                     )
                 )
+            elif sub_args["show"]:
+                data = c.scan.get_net_scan(uuid=sub_args["<uuid>"])
+                log.debug(pformat(data))
+                columns = [
+                    "started_date",
+                    "duration",
+                    "status",
+                    "vulnerabilities_count",
+                    "scanned_hosts",
+                    "name",
+                ]
+                filtered = []
+                for column in columns:
+                    filtered.append(data[column])
```

Comment:
List comprehension this

`filtered = [data[column] for column in columns]`

Reply:
Good catch, thanks!